### PR TITLE
Prevent duplicated wagmi deps

### PR DESCRIPTION
**Pull request by codewiththeflowxyz**
Wagmi has a typescript dep >= 5 and our packages use TS 4.9.3, this was making pnpm duplicate wagmi dependencies in the lock file, one for TS5 and another for TS4.
This isn't usually a problem, but when you work with singletons, react context etc becomes a problem as Context provider need to be the same, if different packaged have different wagmi versions they won't recognize the provider.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "prettier": "prettier --write --cache .",
     "lint": "eslint --cache .",
     "build": "pnpm -r run build",
-    "dev": "pnpm -r --parallel run dev",
+    "dev": "pnpm -r --filter='!./examples/**' --parallel run dev",
```
Comment (generatorpoint) [inline package.json:14]: nice catch

**Comment by generatorpoint** [comment]
should we just upgrade our packages typescript v5?

**Comment by codewiththeflowxyz** [comment]
> should we just upgrade our packages typescript v5?

Yes, that's another option and the first I tried. We can discuss this and decide what is best. I'm up for upgrading.

**Review by bitsforpeace — COMMENTED** [review]
It looks like the install command is failing in CI

**Review by bitsforpeace — APPROVED** [review]

**Pull request merged**